tools/docker/python2.7/Dockerfile

```diff
@@ -59,10 +59,9 @@ ENV PYTHON_VERSION=2.7 \
 
 RUN mkdir ${TUTTLEOFX_BUILD} ${TRAVIS_BUILD_DIR}
 
-COPY . ${TUTTLEOFX_DEV}
+RUN git clone https://github.com/tuttleofx/TuttleOFX.git ${TUTTLEOFX_DEV}
```

Comment:
When you build the Dockerfile, the goal is to build the specific version you are using locally.